02-intermediate-ruby/object-inheritance.md

```diff
@@ -185,7 +189,35 @@ In this case, we utilize the behavior that the _base class_ (`Property`) provide
 
 ![super in method calls](images/inheritance-super-method.png)
 
-**Exercise**  With your seatmates create a `Condo` class.  In addition to the same properties as `Apartment`, a Condo should also have `price` and `square_feet` properties and a `price_per_square_foot` method.
+> **Exercise**: Override the `describe` method in `Circle` to describe radius as well as `color`.
```

Comment:
Should "radius" also be `radius` in order to be consistent with `color`? I can't really tell your intention from this sentence.

Also, I don't think there was ever a concrete description of what the `describe` method should do in the first place, so I think there might be some students that had no idea that the `Shape`'s `describe` method even mentioned `color`.